lib/insights.ts

```diff
@@ -56,6 +57,10 @@ declare global {
   }
 }
 
+type AlgoliaAnalyticsOptions = {
```

Comment:
this isn't reused, so can be inline, but fine like this too

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/5d972fb63234831f94aaed3c625b8d17adc20141